packages/core/test/runSaga.js

```diff
@@ -65,3 +66,58 @@ test('runSaga', assert => {
     })
     .catch(err => assert.fail(err))
 })
+
+test('runSaga with emitter', assert => {
+  assert.plan(1)
+
+  let actual = []
+
+  const em = mitt()
+  let state = null
+  em.on('action', action => (state = action))
+
+  const channel = stdChannel().lift(put => {
```

Comment:
Connecting to an emitter is more straightforward than before as all the connecting logic is here and we do not need `dispatch`.